versioned_docs/version-v0.10/operating/manuals/collaset.md

```diff
@@ -436,4 +436,280 @@ NAME                     READY   STATUS    RESTARTS     AGE
+
+``` shell
+$ kubectl -n default edit cls foo
+ ......
+ spec:
+-  replicas: 0
++  replicas: 2 # scale out 2 pods
+   ......
+```
+
+PVC `foo-www-r4vlh` is retained by Pod `foo-px487` as they have the same `instance-id`. 
+When Pod is created, it searchs PVC with the same instance-id which is owned by CollaSet `foo` and claimed by `volumeClaimTemplates`. 
+The Pod will retain this PVC if the target PVC exists, otherwise create a new one.
+
+``` shell
+$ kubectl -n default edit cls foo
+collaset.apps.kusionstack.io/foo edited
+
+$ kubectl -n default get pod
+NAME        READY   STATUS    RESTARTS   AGE
+foo-ld5xc   1/1     Running   0          27s
+foo-px487   1/1     Running   0          27s
+
+$ kubectl -n default get pvc
+NAME            STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+foo-www-d48gx   Bound    pvc-1884ee45-5cc9-48ee-b01a-20f5ad63d6d4   1Gi        RWO            standard       29s
+foo-www-r4vlh   Bound    pvc-dd7f7cce-a3cb-4bba-a106-e5ad264959a2   1Gi        RWO            standard       2m47s
+
+$ kubectl -n default get pod foo-px487 -o yaml | grep instance-id
+      collaset.kusionstack.io/instance-id: "1"
+
+$ kubectl -n default get pvc foo-www-r4vlh -o yaml | grep instance-id
+      collaset.kusionstack.io/instance-id: "1" # pvc foo-www-r4vlh is retained
+```
+
+#### whenDelete
+Edit foo to configure `Retain` policy fot `whenDelete`, and then delete it.
+``` shell
+$ kubectl -n default edit cls foo
+   ......
+   scaleStrategy:
+     persistentVolumeClaimRetentionPolicy:
+       whenScaled: Retain
++      whenDelete: Retain # retention the pvc after collaset is deleted
+   ......
+collaset.apps.kusionstack.io/foo edited
+
+$ kubectl -n default delete cls foo
+collaset.apps.kusionstack.io "foo" deleted
+```
+
+Recreate `foo` with 2 replicas, and both PVCs are retained.
+``` shell
+$ kubectl -n default apply -f collaset-pvc.yaml
+collaset.apps.kusionstack.io/foo created
+
+$ kubectl -n default get pod
+NAME        READY   STATUS    RESTARTS   AGE
+foo-qhh8t   1/1     Running   0          2s
+foo-ss255   1/1     Running   0          2s
+
+$ kubectl -n default get pvc
+NAME            STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+foo-www-d48gx   Bound    pvc-1884ee45-5cc9-48ee-b01a-20f5ad63d6d4   1Gi        RWO            standard       4m29s
+foo-www-r4vlh   Bound    pvc-dd7f7cce-a3cb-4bba-a106-e5ad264959a2   1Gi        RWO            standard       6m47s
+
+$ kubectl -n default get pod foo-px487 -o yaml | grep instance-id
+      collaset.kusionstack.io/instance-id: "0"
+      collaset.kusionstack.io/instance-id: "1"
+
+$ kubectl -n default get pvc foo-www-r4vlh -o yaml | grep instance-id
+      collaset.kusionstack.io/instance-id: "0" # pvc foo-www-d48gx is retained
+      collaset.kusionstack.io/instance-id: "1" # pvc foo-www-r4vlh is retained
+```
+
+#### Update VolumeClaimTemplates
```

Comment:
I think this section should be the second one before section 'PVC Retention Policy'